### PR TITLE
Create OpenAPIV3 schema for antrea versions ( 1.2.3, 0.13.3, 0.11.3 ) and generate package with the schema

### PR DESCRIPTION
**Pull request by shivaani0505**
## What this PR does / why we need it

- [x] Create OpenAPIV3 schema for antrea versions ( 1.2.3, 0.13.3, 0.11.3 ) and generate package with the schema

## Details for the Release Notes (PLEASE PROVIDE)

```release-note
Create OpenAPIV3 schema for antrea package and generate package with the schema
```

## Which issue(s) this PR fixes

Fixes: #2842 

## Describe testing done for PR

Tested with following ytt commands locally:
<version> below can be substituted with each item from the list: ( 1.2.3, 0.13.3, 0.11.3 )
* make generate-openapischema-package PACKAGE=antrea VERSION=1.2.3
* make push-package PACKAGE=antrea VERSION=1.2.3 TAG=1.2.3

## Special notes for your reviewer

ytt version => 0.38.0
kapp version => 0.37.0
yq version => 4.11.1

Also, I would be adding github workflow in the same PR which I am currently working on

### CODE REVIEW

**Review by blc1996 — APPROVED** [review]
LGTM

**Comment by blc1996** [comment]
Hi @vmware-tanzu/pkg-antrea-owners , could someone also review the added openAPI schema for Antrea? Thanks!

File: addons/packages/antrea/0.11.3/package.yaml [hunk 1]
```diff
@@ -5,24 +5,89 @@ metadata:
...
-    - "Apache 2.0"
+  - Apache 2.0
   template:
     spec:
       syncPeriod: 5m
       fetch:
-        - imgpkgBundle:
-            image: projects.registry.vmware.com/tce/antrea@sha256:35a2848a2e6cb317f40776e16b2511b226d11930a5245747b655678e5b9a65a1
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/tce/antrea@sha256:26404f38371f0e38c8997c302d5442387763346965f6d1d112535c8bd44fd4da
```
Comment (knabben) [inline addons/packages/antrea/0.11.3/package.yaml:16]: why the hash is being changed here?

File: addons/packages/antrea/0.13.3/README.md [hunk 2]
```diff
@@ -12,7 +12,6 @@ The following configuration values can be set to customize the antrea installati
 
 | Value | Required/Optional | Description |
 |-------|-------------------|-------------|
-| `namespace` | Optional | The namespace in which to deploy antrea. |
```
Comment (knabben) [inline addons/packages/antrea/0.13.3/README.md:15]: isn't namespace not a field, why removing?

**Comment by knabben** [comment]
LGTM

**Review by knabben — APPROVED** [review]

**Comment by shivaani0505** [comment]
> LGTM

Could you provide your approval @knabben

**Pull request merged**